tests/director/tests/formation_api_test.go

```diff
@@ -626,7 +625,9 @@ func TestRuntimeContextsFormationProcessingFromASA(stdT *testing.T) {
 			providerRuntimeInput := graphql.RuntimeRegisterInput{
 				Name:        "providerRuntime",
 				Description: ptr.String("providerRuntime-description"),
-				Labels:      graphql.Labels{conf.SubscriptionConfig.SelfRegDistinguishLabelKey: conf.SubscriptionConfig.SelfRegDistinguishLabelValue, tenantfetcher.RegionKey: conf.SubscriptionConfig.SelfRegRegion},
+				Labels: graphql.Labels{
```

Comment:
Is there any documentation that we need to update either open source or some internal wiki, regarding stakeholders that use these flows and no longer need to provide region label?

Reply:
Yes, I have in mind the App Provider onboarding wiki, which will be updated after this change is merged and ensured with the next release.